examples/terminal/terminal

```diff
@@ -50,26 +50,37 @@ MAIN_DIALOG='
 				<action signal="child-exited">echo "vte0=$vte0 - New PID"</action>
 				<action signal="show">echo "vte0=$vte0 - show signal"</action>
 			</terminal>
-			<terminal argv0="/bin/sh"
+			<terminal argv0="'${SHELL:-/bin/sh}'"
 				envv0="WELCOME=Welcome to the terminal (VteTerminal) widget :)"
 				pointer-autohide="true"
 				current-directory-uri="'$HOME'"
-				cursor-background-color="#ff0"
-				highlight-background-color="#660"
-				text-background-color="#332"
-				text-foreground-color="#fff"
+
+				cursor-background-color="aqua"
+				highlight-background-color="#808000"
+				highlight-foreground-color="red"
+				text-background-color="#44d"
+				text-foreground-color="#ddd"
+
 				hscrollbar-policy="2"
 				vscrollbar-policy="2">
 				<variable>vte1</variable>
-				<input>echo echo \$WELCOME</input>
+
+				<input>echo "_info() { printf \"\033[48;2;%d;%d;%dm     \033[0m %s\n\" \"\$@\"; }"</input>
+				<input>echo " echo;echo;echo; echo \"\$WELCOME\"; echo;echo;echo"</input>
+				<input>echo "_info   0 255 255 \"cursor background color #0ff (aqua)\""</input>
+				<input>echo "_info 128 128   0 \"text select background color #222 (olive)\""</input>
+				<input>echo "_info 255   0   0 \"text select foreground color #f00 (red) GTK3 only\""</input>
+				<input>echo "_info  68  68 221 \"text background color #44d (blue)\""</input>
+				<input>echo "_info 221 221 221 \"text foreground color #ddd (gray)\""</input>
```

Comment:
Thank you for your comment.  Let me explain what I had in mind when I wrote that block. In gtk3 the block produces the output at the bottom of this screenshot, where each color has a corresponding color "tile" with a label describing which color property is affected. Each label ends with the name of the color in parens.

![gtk3-20220307](https://user-images.githubusercontent.com/1414102/157122345-bf7ba2ce-9231-45fe-bf4c-9ce17950a424.png)

In gtk2 the same block produces essentially the same output with the exception of the "tiles".  That's because VTE for gtk2 does not implements the truecolor ANSI codes that I use to create the "tiles".

![gtk2-20220307](https://user-images.githubusercontent.com/1414102/157122363-270043a9-fae7-4942-b274-abd69accdc7f.png)

All considered, I think the block is useful for both gtk3 and gtk2 users, and I would rather leave it as is.

Reply:
Sure, let it be, I was just thinking about a note that the tiles show up only with GTK3.